packages/web-app-files/src/components/SideBar/Shares/Collaborators/CollaboratorsEditOptions.vue

```diff
@@ -335,6 +381,34 @@ export default {
       return role.name === 'advancedRole'
     },
 
+    confirmCustomPermissions() {
+      this.$refs.customPermissionsDrop.hide()
```

Comment:
UX wise, what do you think about disabling the `Apply` button when no permissions are checked and additionally having an early return here if `this.customerPermissions` is empty, just to be sure?

Reply:
What if the user would like to share only with read permission? We could do this earlier thanks to viewer role having additional checkbox for sharing but since that's gone I don't see any other option how to do it without creating a new role :shrug:

Reply:
Hah lol. Review fail. But: this shows me a UX issue. Could you statically add a disabled and checked `read` checkbox? So that it's clear that read permissions are always included? Otherwise the custom permissions with all checkboxes disabled give the impression that you are creating a share without any permission whatsoever.

Reply:
Sounds good :+1:

Reply:
Added it